skimage/metrics/_structural_similarity.py

```diff
@@ -72,7 +77,8 @@ def structural_similarity(
     Returns
     -------
     mssim : float
-        The mean structural similarity index over the image.
+        The mean structural similarity index over the SSIM image. If present in the
+        image, edge artefacts from padding are ignored in the computation.
```

Comment:
```suggestion
        image, edge artifacts from padding are ignored in the computation.
```
(we use US spelling everywhere)